server/game/cards/06-WoE/Outnegotiate.js

```diff
@@ -0,0 +1,23 @@
+const Card = require('../../Card.js');
+
+class Outnegotiate extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            target: {
+                controller: 'opponent',
+                location: 'discard',
+                gameAction: [
+                    ability.actions.steal((context) => ({
+                        target: context.target.owner,
+                        amount: context.target.bonusIcons.length
+                    })),
+                    ability.actions.returnToDeck({ bottom: true })
```

Comment:
Having log issues again here; I don't see the return to deck message get printed in the log.

Reply:
Because it is an array of events, there is a flag that allows multiple messages to be displayed. 
Add `effectStyle: all` at the `target` level.

Reply:
Thanks, done!